com.unity.render-pipelines.universal/ShaderLibrary/RealtimeLights.hlsl

```diff
@@ -131,6 +134,12 @@ Light GetMainLight(float4 shadowCoord, float3 positionWS, half4 shadowMask)
 {
     Light light = GetMainLight();
     light.shadowAttenuation = MainLightShadow(shadowCoord, positionWS, shadowMask, _MainLightOcclusionProbes);
+
+    #ifdef _MAIN_LIGHT_COOKIE
+        half3 cookieColor = URP_LightCookie_SampleMainLightCookie(positionWS);
```

Comment:
This line will introduce a shader warning here when real is defined as float since the function returns a real3.

Reply:
Yes, thanks. It seems I've forgotten replace it.